references/segmentation/transforms.py

```diff
@@ -75,14 +74,23 @@ def __call__(self, image, target):
         return image, target
 
 
-class ToTensor(object):
+class PILToTensor:
     def __call__(self, image, target):
         image = F.pil_to_tensor(image)
-        image = F.convert_image_dtype(image)
         target = torch.as_tensor(np.array(target), dtype=torch.int64)
         return image, target
 
 
+class ConvertImageDtype:
+    def __init__(self, dtype):
+        self.dtype = dtype
+
+    def __call__(self, image, target):
+        image = F.convert_image_dtype(image, self.dtype)
+        target = torch.as_tensor(target, dtype=torch.int64)
```

Comment:
Why don't we let target unchanged here? Aka straight return it without modifications?

Reply:
Yes, you are right. This is not needed. Thanks!